src/app/seamlyme/dialogs/dialogmdatabase.cpp

```diff
@@ -423,93 +423,162 @@ void DialogMDataBase::Recheck()
-    InitGroup(&groupH, "H. " + tr("Vertical", "Measurement section"),                   ListGroupH(), list);
-    InitGroup(&groupI, "I. " + tr("Horizontal", "Measurement section"),                 ListGroupI(), list);
-    InitGroup(&groupJ, "J. " + tr("Bust", "Measurement section"),                       ListGroupJ(), list);
-    InitGroup(&groupK, "K. " + tr("Balance", "Measurement section"),                    ListGroupK(), list);
-    InitGroup(&groupL, "L. " + tr("Arm", "Measurement section"),                        ListGroupL(), list);
-    InitGroup(&groupM, "M. " + tr("Leg", "Measurement section"),                        ListGroupM(), list);
-    InitGroup(&groupN, "N. " + tr("Crotch and Rise", "Measurement section"),            ListGroupN(), list);
-    InitGroup(&groupO, "O. " + tr("Men & Tailoring", "Measurement section"),            ListGroupO(), list);
-    InitGroup(&groupP, "P. " + tr("Historical & Specialty", "Measurement section"),     ListGroupP(), list);
-    InitGroup(&groupQ, "Q. " + tr("Patternmaking measurements", "Measurement section"), ListGroupQ(), list);
-}
-
+    InitGroup(&groupA, groupAText, ListGroupA(), newList);
+    InitGroup(&groupB, groupBText, ListGroupB(), newList);
+    InitGroup(&groupC, groupCText, ListGroupC(), newList);
+    InitGroup(&groupD, groupDText, ListGroupD(), newList);
+    InitGroup(&groupE, groupEText, ListGroupE(), newList);
+    InitGroup(&groupF, groupFText, ListGroupF(), newList);
+    InitGroup(&groupG, groupGText, ListGroupG(), newList);
+    InitGroup(&groupH, groupHText, ListGroupH(), newList);
+    InitGroup(&groupI, groupIText, ListGroupI(), newList);
+    InitGroup(&groupJ, groupJText, ListGroupJ(), newList);
+    InitGroup(&groupK, groupKText, ListGroupK(), newList);
+    InitGroup(&groupL, groupLText, ListGroupL(), newList);
+    InitGroup(&groupM, groupMText, ListGroupM(), newList);
+    InitGroup(&groupN, groupNText, ListGroupN(), newList);
+    InitGroup(&groupO, groupOText, ListGroupO(), newList);
+    InitGroup(&groupP, groupPText, ListGroupP(), newList);
+    InitGroup(&groupQ, groupQText, ListGroupQ(), newList);
+}                                  
+                                   
 //---------------------------------------------------------------------------------------------------------------------
-void DialogMDataBase::InitGroup(QTreeWidgetItem **group, const QString &groupName, const QStringList &mList,
-                                const QStringList &list)
-{
-    *group = AddGroup(groupName);
-    for (int i=0; i < mList.size(); ++i)
-    {
-        AddMeasurement(*group, mList.at(i), list);
-    }
-}
-
+void DialogMDataBase::InitGroup(QTreeWidgetItem **group, const QString &groupName, const QStringList &groupList,
+                                const QStringList &newList)
+{                                  
+    *group = AddGroup(groupName);  
+    for (int i=0; i < groupList.size(); ++i)
```

Comment:
Thanks for renaming mList as groupList.

Reply:
setSoapBox(on);

Why stop at renaming mList as groupList... if keeping with Qt style convention and for readability it should be MeasurementsDatabaseDialog not DialogMDataBase. 

The code is rife with nonsensical abbreviations that should NEVER be used... especially in open source where it's important for multiple contributors actually being able to read the code.  For example what is a MDataBase dialog? MeasurementsDatabaseDialog leaves no question. Also by the Qt (English left to right) naming style the class name (Dialog) should be on the right. 

setSoapBox(off);

Reply:
@DSCaskey

> setSoapBox(on);
> 
> Why stop at renaming mList as groupList... if keeping with Qt style convention and for readability it should be MeasurementsDatabaseDialog not DialogMDataBase.
> 
> The code is rife with nonsensical abbreviations that should NEVER have been used... especially in open source where it's important for multiple contributors actually being able to read the code. For example what is a MDataBase dialog? MeasurementsDatabaseDialog leaves no question. Also by the Qt (English left to right) naming style the class name (Dialog) should be on the right.
> 
> setSoapBox(off);

You have a good point - However, I think to avoid going overboard, making too many readability/naming changes is a bit out of scope for this PR. Always makes sense to refactor this kind of stuff as work gets done throughout the project, though. I'll make a few more changes in this realm for this PR to make a dent in what needs to be done. Good suggestion.